wiki/en/misc/1-index.html

```diff
@@ -30,26 +30,25 @@ <h2>What is Jamulus?</h2>
 </div>
 <div class="fx-row fx-row-center-xs">
   <div class="fx-col-100-xs fx-col-50-l">
-     <h2>Jamulus worldwide</h2>
-    All over the world Jamulus allows huge choirs to rehearse, loud rock bands to play, and brings folk and classical musicians together. It's being used for remote music lessons,
-    in schools and universities, in private and in public. Even just for the spoken word – all in real time on the Internet, as if you were there in person.
-    <h2>Help needed?</h2>
+     <h2>Worldwide</h2>
+    Huge choirs, rock bands, folk and classical musicians, remote music lessons, in schools and universities,
```

Comment:
```suggestion
    Rehersals, tutoring, performances, and even the spoken word,
```

Reply:
The heading "Worldwide" doesn't seem connected to the text any more.

Reply:
The same was true before?
It was "Jamulus worldwide
All over the world Jamulus allows …" That it happens on the Internet is already established by "via broadband".

The thinking is "as if you were there in person …", pointing to ~"the whole world".
The world comes to you, and you can be anywhere.